models/marts/documentation/fct_undocumented_sources.sql

```diff
@@ -0,0 +1,22 @@
+with
+
+all_resources as (
+    select * from {{ ref('int_all_graph_resources') }}
+    where not is_excluded
+
+),
+
+final as (
+
+    select
+        resource_name,
```

Comment:
per the above, we can use `all_resource.source_name`, then make this a `distinct`

Reply:
Brilliant - thank you